scripts/Install-VSCode.ps1

```diff
@@ -1,6 +1,6 @@
 <#PSScriptInfo
 
-.VERSION 1.3
+.VERSION 1.3.1
```

Comment:
Since you're adding a feature, this would be 1.4.0. Can you update that?

Reply:
Done in 96adddaba9960f9ee9ad75cce20f84be4edc6213